modules/webgl/src/classes/program.js

```diff
@@ -132,7 +131,7 @@ export default class Program extends Resource {
       this.setUniforms(uniforms || {});
     }
 
-    if (this.debug && logPriority !== undefined) {
+    if (log.priority >= 2 && logPriority !== undefined) {
```

Comment:
This should be `log.priority >= logPriority`?

Reply:
Done.